src/index.js

```diff
@@ -4,6 +4,7 @@ import * as ThemeUI from 'theme-ui';
 export { default as theme } from './components/theme';
 
 // Components
+export { default as Modal } from './components/modal';
```

Comment:
We might want to export ReactModal from the library as well. In case we need to prototype a new modal in the fng-web-hardware project

Reply:
I've added `ReactModal` to the exports.